tosca/validate-evmzero.jenkinsfile

```
@@ -42,10 +42,9 @@ pipeline {
                 		branches: [[name: "${ToscaVersion}"]],
                 		userRemoteConfigs: [[url: 'https://github.com/Fantom-foundation/Tosca.git']]
                 	)
+                	sh "git submodule update --recursive"
```

```suggestion
                	sh "git submodule update --recursive --depth 1"
```
To avoid a complete clone and download only the head of the main branch.